PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -21,16 +21,38 @@
         android:textColor="@android:color/black"
         android:textSize="@dimen/game_text_size" />
 
-    <ListView
-        android:id="@+id/mainListView"
-        android:layout_width="@dimen/main_list_view_width"
-        android:layout_height="@dimen/main_list_view_height"
+    <ImageView
+        android:id="@+id/askerImageView"
+        android:layout_width="@dimen/asker_image_view_width"
+        android:layout_height="@dimen/asker_image_view_height"
         android:layout_alignParentRight="true"
-        android:layout_below="@+id/askerImageView"
-        android:background="@android:color/white"
-        android:backgroundTint="@color/abc_secondary_text_material_dark"
-        android:clickable="true" >
-    </ListView>
+        android:layout_below="@id/scenarioNameEditText"
+        android:src="@drawable/marco" />
+
+
+    <LinearLayout
+        android:layout_width="250dp"
```

Comment:
Don't use absolute values. Define dimensions on dimens.xml and refer to them using `@dimen/dimension_name`